Emby.Dlna/Main/DlnaEntryPoint.cs

```diff
@@ -276,8 +282,9 @@ private async Task RegisterServerEndpoints()
 
                 _logger.LogInformation("Registering publisher for {0} on {1}", fullService, address);
 
-                var descriptorUri = "/dlna/" + udn + "/description.xml";
-                var uri = new Uri(_appHost.GetLocalApiUrl(address) + descriptorUri);
+                var descriptorUri = (_config.Configuration.BaseUrl ?? string.Empty + "/dlna/" + udn + "/description.xml").TrimStart('/');
+
+                var uri = new Uri(string.IsNullOrEmpty(_publishedServerUrl) ? _appHost.GetLocalApiUrl(address) + "/" + descriptorUri : _publishedServerUrl + descriptorUri);
```

Comment:
I am not a fan of ternaries in string concatenation.

Reply:
extracted out